package.json

```diff
@@ -115,7 +116,7 @@
     "cypress-pipe": "^1.5.0",
     "cypress-testing-library": "^2.3.4",
     "cypress-wait-until": "^1.3.0",
-    "ganache-cli": "^6.9.1",
+    "ganache-cli": "6.12.2",
```

Comment:
would be awesome if you leave the caret here, so we can get the future bug fixes.

Reply:
Sure good point